terraform/variables.tf

```diff
@@ -0,0 +1,22 @@
+
+variable "unique_suffix" {
```

Comment:
IMO you should not use this. We use it in marsha to prevent existing bucket deletion, it wasn't an option for us to delete the production bucket but as ashley is still not using aws in production yet you should use the `bucket_prefix` option.

Reply:
Thanks, I forgot this.